pastas/io/pas.py

```diff
@@ -86,5 +86,7 @@ def default(self, o):
             return o.to_timedelta64().__str__()
         elif isna(o):
             return None
+        elif isinstance(obj, (np.integer)):
```

Comment:
The proposed change is inconsistent in the variable name (`obj `instead of `o`). Can you change the variable name?

Also, brackets around np.integer are redundant and should be removed.